src/domain/DailyPoolData.ts

```diff
@@ -40,6 +40,7 @@ export function createDailySnapshot(block: ethereum.Block): void {
     updateRewardDayTotal(yesterdayTimeStamp, pool)
     calculateRewards(yesterdayTimeStamp, pool)
   }
+  resetActiveInvestments()
```

Comment:
i think that hasActiveInvestment should only be used by the rewards calculation and if we want to know for the user we should query their token balances - if i am understanding you correctly

Reply:
Agree that it should not be used for a different purpose than you describe, but since the graph is exposing all stored data publicly, I'd strongly argue for consistent data. Right now, you will get something like: 

```gql
{
  account(id:"0x10ac55ef72d6301236b315fe8d60ca6af7a4628c") {
    id
    hasActiveInvestment
  }
  tokenBalances(where: {owner: "0x10ac55ef72d6301236b315fe8d60ca6af7a4628c"}) {
    value
  }
}
```

```json
{
  "data": {
    "account": {
      "hasActiveInvestment": false,
      "id": "0x10ac55ef72d6301236b315fe8d60ca6af7a4628c"
    },
    "tokenBalances": [
      {
        "value": "25670398849801977013430"
      }
    ]
  }
}
```

Which is wrong, since there is an active investment.

Reply:
We could also use `hasActiveInvestment` internally only, but I feel we should not store it then and just pass it to other functions as an argument (e. g. a map from account to active investment bool).